### PR TITLE
XRUBY_LIB and XRUBY_ROOT have been changed to EXERCISM_RUBY_LIB and E…

### PR DESCRIPTION
**Pull request by NeimadTL**
XRUBY_LIB and XRUBY_ROOT have been changed to EXERCISM_RUBY_LIB and EXERCISM_RUBY_ROOT;

gitter url in 24pullrequests.md has been updated

### CODE REVIEW

**Comment by Insti** [comment]
Hi, Thanks, Welcome.
I had a quick look and this looks fine.
I'd like to download and run it to double check before I merge it, I'll do that sometime in the next day or so.

**Comment by NeimadTL** [comment]
Hi, you're welcome.

no problem. I'd like to know how you run it too so that I can run myself locally before committing for next time.

**Comment by Insti** [comment]
Try starting here: https://github.com/exercism/ruby#regenerating-a-test-suite
And generating a few test suites.

**Comment by NeimadTL** [comment]
Thank you

File: docs/24pullrequests.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ Something missing?  Create a pull request!
 
 Check out the [Getting Involved in a Track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md) documentation, and some [ideas for things we are looking for](http://exercism.io/languages/ruby/todo) for exercises that are needed to be implemented.
 
-We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/xruby](https://badges.gitter.im/exercism/xruby.svg)](https://gitter.im/exercism/xruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/xruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```
Comment (kotp) [inline docs/24pullrequests.md]: Can you rebase against master?  I think these changes are already in place, and rebasing on master should end up removing this change from this PR.

Comment (NeimadTL) [reply docs/24pullrequests.md]: I did and unless I made a mistake, the changes wasn't in place.

Comment (kotp) [reply docs/24pullrequests.md]: It was me misidentifying the file as README in project root.  So not used to looking at these in the browser.  When I finally checked it locally, it was glaring me right in the eye.

**Comment by NeimadTL** [comment]
just made some changes.
Now, in  test/fixtures there is a directory called xruby...I'm about to change it.
Thank you guys for your feedback @kotp @Insti

**Comment by Insti** [comment]
I think you got them all now @NeimadTL, thanks for all your work on this. :heart:

**Comment by NeimadTL** [comment]
you're welcome. Thank you for your help and support @Insti @kotp.
Can you suggest me another issue number to work on please (As I don't really know the project) ?

**Comment by Insti** [comment]
#386 maybe?

**Comment by NeimadTL** [comment]
I'll check it out...thank you

**Comment by Insti** [comment]
I also found a few more xruby references: https://github.com/exercism/ruby/issues/680#issuecomment-325204762

**Comment by NeimadTL** [comment]
just pushed a correction for them

**Pull request merged**